charts/aad-pod-identity/README.md

```diff
@@ -0,0 +1,97 @@
+* AzureIdentity `CustomResourceDefinition`
+* AzureIdentityBinding `CustomResourceDefinition`
+* AzureAssignedIdentity `CustomResourceDefinition`
+* AzureIdentity instance
+* AzureIdentityBinding instance
+* Managed Identity Controller (MIC) `Deployment`
+* Node Managed Identity (NMI) `DaemonSet`
+
+## Getting Started
+The following steps will help you create a new Azure identity ([Managed Service Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) or [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals)) and assign it to pods running in your Kubernetes cluster.
+
+### Prerequisites
+* [Azure Subscription](https://azure.microsoft.com/)
+* [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/services/kubernetes-service/) or [ACS-Engine](https://github.com/Azure/acs-engine) deployment
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (authenticated to your Kubernetes cluster)
+* [Helm v1.10+](https://github.com/helm/helm)
+* [Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [git](https://git-scm.com/downloads)
+
+### Steps
+
+1. Create a new [Azure User Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) using the Azure CLI:
+> __NOTE:__ It's simpler to use the same resource group as your Kubernetes nodes are deployed in. For AKS this is the MC_* resource group. If you can't use the same resource group, you'll need to grant the Kubernetes cluster's service principal the "Managed Identity Operator" role.
+```shell
+az identity create -g <resource-group> -n <id-name>
+```
+
+2. Assign your newly created identity the role of _Reader_ for the resource group:
+```shell
+az role assignment create --role Reader --assignee <principal-id> --scope /subscriptions/<subscription-id>/resourcegroups/<resource-group>
+```
+
+3. Clone this repository and navigate to the helm chart's directory.
+```shell
+git clone git@github.com:Azure/aad-pod-identity.git && cd aad-pod-identity/charts/aad-pod-identity
+```
+
+4. Open the `values.yaml` file in a text editor.
+
+5. Update the `azureIdentity` values with your Azure identity's resource ID and client ID (retrievable from the CLI or portal).
+
+6. Update the `azureIdentityBinding` selector value with a value that will match the label applied to the pods you wish to assign the identity to i.e. `selector: demo`.
+
+7. Ensure you have helm initialized correctly to work with your cluster. If not, follow this [guide](https://docs.helm.sh/using_helm/#initialize-helm-and-install-tiller). If your cluster has rbac-enabled, you'll need to initialize tiller with a suitable `service-account`.
+
+8. Install the helm chart into your Kubernetes cluster using your updated `values.yaml`.
+```shell
+helm install --values values.yaml .
```

Comment:
@jjcollinge - one last request (sorry for the late comment) - could we document a flow where only the infra pieces are deployed, without any identities being created. Was hoping this can be leveraged for other projects who run on top of this project (eg: app gw) and will add the identities and binding as per their need.

Reply:
@kkmsft I've added a flag so you can enable/disable the azure identity and binding which should satisfy your requirement. I've not updated the README.md to explicitly call this out as I didn't want to complicate the sequential getting started steps with different branches.